### PR TITLE
delete nav names/add subnav dropdown to gotchis/push up nav container

### PR DESCRIPTION
**Pull request by npankovich**

### CODE REVIEW

File: src/components/custom/CustomTooltip.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { styled, Tooltip, tooltipClasses } from '@mui/material';
 
-interface CustomTooltipProps {
+export interface CustomTooltipProps {
```
Comment (dmutrokot) [inline src/components/custom/CustomTooltip.tsx:3]: please move this interface into `shared`, due to it's use there

**Review by dmutrokot — CHANGES_REQUESTED** [review]

File: src/components/custom/CustomTooltip.tsx (hunk 1 above)
Comment (npankovich) [reply src/components/custom/CustomTooltip.tsx]: changed/thnxs

**Review by dudendy — APPROVED** [review]

File: src/shared/models/page-nav.model.ts [hunk 2]
```diff
@@ -1,3 +1,4 @@
+import { CustomTooltipProps } from 'shared/models';
```
Comment (dmutrokot) [inline src/shared/models/page-nav.model.ts:1]: please add whitespace after import

File: src/shared/models/index.ts [hunk 3]
```diff
@@ -23,3 +23,4 @@ export * from './tile.model';
 export * from './token.model';
 export * from './traits.model';
 export * from './wearable.model';
+export * from './custom-tooltip.model';
```
Comment (dmutrokot) [inline src/shared/models/index.ts:26]: please move to related alphabetical imports

File: src/components/custom/CustomTooltip.tsx [hunk 4]
```diff
@@ -1,37 +1,32 @@
...
-  [`& .${tooltipClasses.tooltip}`]: {
-    backgroundColor: theme.palette.secondary.dark,
+  const onToggleTooltip = (value: boolean): void => {
+    setIsTolltipOpened(value);
+  };
+
+  return <div
+    onClick = {() => onTooltipClick()}
+    onMouseEnter={() => onToggleTooltip(true)}
+    onMouseLeave={() => onToggleTooltip(false)}>
```
Comment (dmutrokot) [inline src/components/custom/CustomTooltip.tsx:20]: `>` should be in new line

File: src/components/custom/CustomTooltip.tsx [hunk 5]
```diff
@@ -1,37 +1,32 @@
...
+
```
Comment (dmutrokot) [inline src/components/custom/CustomTooltip.tsx:21]: no need for white space here

File: src/components/custom/CustomTooltip.tsx [hunk 6]
```diff
@@ -1,37 +1,32 @@
...
+    <Tooltip {...props} open={isTolltipOpened} classes={{ popper: className }}/>
+      </div>;
```
Comment (dmutrokot) [inline src/components/custom/CustomTooltip.tsx:23]: please indent properly

File: src/components/custom/CustomTooltip.tsx [hunk 7]
```diff
@@ -1,37 +1,32 @@
...
-    | 'top';
-  disableInteractive?: boolean;
-  followCursor?: boolean;
-  enterTouchDelay?: number;
-  className?: string;
-  arrow?: boolean;
-  open?: boolean;
-}
+const StyledTooltip = styled(({ className, ...props }: CustomTooltipProps) => {
+
```
Comment (dmutrokot) [inline src/components/custom/CustomTooltip.tsx:6]: no need whitespace here, whitespaces needed when they logically separates something,
e.g.
`
const classes = styles();

const [someState, setSomeState] = useState<T>(initialValueOfTypeT);
`

File: src/components/custom/CustomTooltip.tsx [hunk 8]
```diff
@@ -1,37 +1,32 @@
+import { useState } from 'react';
 import { styled, Tooltip, tooltipClasses } from '@mui/material';
+import { CustomTooltipProps } from 'shared/models';
```
Comment (dmutrokot) [inline src/components/custom/CustomTooltip.tsx:3]: we have following rules about imports : https://www.notion.so/fireball-gg/General-41d2065755d949f78a1438b58acf328c

**Review by dmutrokot — CHANGES_REQUESTED** [review]

File: src/shared/models/page-nav.model.ts (hunk 2 above)
Comment (npankovich) [reply src/shared/models/page-nav.model.ts:1]: thxs/done

File: src/components/custom/CustomTooltip.tsx (hunk 4 above)
Comment (npankovich) [reply src/components/custom/CustomTooltip.tsx]: thxs/done

File: src/components/custom/CustomTooltip.tsx (hunk 5 above)
Comment (npankovich) [reply src/components/custom/CustomTooltip.tsx]: thxs/done

File: src/components/custom/CustomTooltip.tsx (hunk 6 above)
Comment (npankovich) [reply src/components/custom/CustomTooltip.tsx]: thxs/done

File: src/components/custom/CustomTooltip.tsx (hunk 7 above)
Comment (npankovich) [reply src/components/custom/CustomTooltip.tsx]: thxs/done

File: src/components/custom/CustomTooltip.tsx (hunk 8 above)
Comment (npankovich) [reply src/components/custom/CustomTooltip.tsx:4]: thxs/done

File: src/components/CustomDropdown/CustomDropdown.tsx [hunk 9]
```diff
@@ -0,0 +1,74 @@
+import { useState } from 'react';
```
Comment (dmutrokot) [inline src/components/CustomDropdown/CustomDropdown.tsx:1]: do we need this component?

File: src/shared/models/page-nav.model.ts [hunk 10]
```diff
@@ -6,6 +6,7 @@ export interface PageNavLink {
   count?: number | string;
   isShowSubRoutes?: boolean;
   subNavComponent?: JSX.Element;
+  dropdown?: boolean;
```
Comment (dmutrokot) [inline src/shared/models/page-nav.model.ts:9]: seems to be redundant property, you can handle how to show/or not show with `isShowSubRoutes`

File: src/shared/models/custom-tooltip.model.ts [hunk 11]
```diff
@@ -0,0 +1,23 @@
+export interface CustomTooltipProps {
```
Comment (dmutrokot) [inline src/shared/models/custom-tooltip.model.ts:1]: if `StyledDropdown` from comment above is not needed, please move it back to `CustomTooltip` component

**Review by dmutrokot — CHANGES_REQUESTED** [review]

File: src/components/CustomDropdown/CustomDropdown.tsx (hunk 9 above)
Comment (npankovich) [reply src/components/CustomDropdown/CustomDropdown.tsx:1]: no/delete

File: src/shared/models/custom-tooltip.model.ts (hunk 11 above)
Comment (npankovich) [reply src/shared/models/custom-tooltip.model.ts:1]: done/thnxs

File: src/shared/models/page-nav.model.ts (hunk 10 above)
Comment (npankovich) [reply src/shared/models/page-nav.model.ts:9]: done/thnxs

**Review by dudendy — APPROVED** [review]

**Review by dudendy — CHANGES_REQUESTED** [review]
seems like explorer page is broken in the current branch
https://deploy-preview-510--melodious-elf-4231c5.netlify.app/explorer?sort=mrs&dir=desc
Screenshot 2023-03-14 at 12 51 31

**Comment by dudendy** [comment]
> seems like explorer page is broken in the current branch https://deploy-preview-510--melodious-elf-4231c5.netlify.app/explorer?sort=mrs&dir=desc Screenshot 2023-03-14 at 12 51 31

resolved by @dmutrokot

**Review by dudendy — APPROVED** [review]

**Review by dmutrokot — APPROVED** [review]

**Pull request merged**